ot3_state_manager/setup.py

```diff
@@ -0,0 +1,72 @@
+# Inspired by:
```

Comment:
setup.py should be replaceable with pyproject toml configuration: https://setuptools.pypa.io/en/latest/userguide/pyproject_config.html